Moovies/Moovies/DescriptionModule/Presentor/DescriptionPresenter.swift

```diff
@@ -0,0 +1,42 @@
+// DescriptionPresenter.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import UIKit
+
+protocol DescriptionViewProtocol: AnyObject {
+    func reloadTable()
+}
+
+protocol DescriptionViewPresentorProtocol: AnyObject {
+    var details: Description { get }
+    var ide: Int { get }
+    func getMoovieDescription()
+}
+
+class DescriptionPresentor: DescriptionViewPresentorProtocol {
+    private var movieAPIservice: MovieAPIServiceProtocol
+    weak var view: DescriptionViewProtocol?
+    var details: Description
```

Comment:
```suggestion
    var details: Description
    var ide: Int
```
Выше private перенеси